### PR TITLE
Add images to cards

### PR DESCRIPTION
**Pull request by patricksimonian**
# About
added images to all card components

## Changes
- added images to cards
- restyled cards so that images fit predictably in containers when they exist
- added truncation of description text incase it's over 5 lines

### CODE REVIEW

**Comment by patricksimonian** [comment]
not ready for merge until #143 is merged and **this pr is rebased ontop of master**

File: app-web/src/components/Cards/Card/ComponentCard.js [hunk 1]
```diff
@@ -39,6 +45,11 @@ ComponentCard.propTypes = {
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
   resourcePath: PropTypes.string.isRequired,
+  image: PropTypes.string,
+};
+
+ComponentCard.defaultProps = {
+  image: '',
```
Comment (sutherlanda) [inline app-web/src/components/Cards/Card/ComponentCard.js]: Maybe set a default image path here? You could make the default image a configuration item like you did with maxTitleLines, maxDescription lines.

Comment (patricksimonian) [reply app-web/src/components/Cards/Card/ComponentCard.js]: yes! Thats a great idea, will be working on that on a new pull request. We are going to first identify what default images to use based on the card type :)

**Comment by ShellyXueHan** [comment]
sourcePath for card components is required

File: app-web/src/components/Cards/Card/ComponentCard.js (hunk 1 above)
Comment (patricksimonian) [reply app-web/src/components/Cards/Card/ComponentCard.js]: The react `<Image>` component will not render a 'broken image' if the image fails to load which is the fall back until the next PR

**Pull request merged**